### PR TITLE
Explain how to set a manual mask

### PR DESCRIPTION
**Pull request by zeroheure**
Add configurations places in Readme's Configuration part. Change paragraphs sorting.

### CODE REVIEW

**Review by andreampiovesana — APPROVED** [review]
go

**Review by TheMule71 — APPROVED** [review]
Improved instructions.

**Review by ivantodorovich — APPROVED** [review]

File: product_variant_default_code/README.rst [hunk 1]
```diff
@@ -28,19 +28,27 @@ variants references are automatically set. For example:
...
+and look for `default_reference_missing_placeholder` and `default_reference_separator`.
+
+To set the reference mask  on any product template, go to *Settings > General 
+Settings* and in "Product Default Code Behaviour" choose the "Manual Mask" 
+option. Then the 'Variant reference mask' field will be visible on product 
+template, *after product creation*. In this field, put the *Attribute Name* 
+inside `[]` marks (it is case sensitive).
 
 Example:
+--------
```
Comment (pedrobaeza) [inline product_variant_default_code/README.rst:51]: Non valid title level

```suggestion
~~~~~~~~~
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You can take the occasion to split the README in parts in `readme` folder for better maintenance.

File: product_variant_default_code/README.rst (hunk 1 above)
Comment (zeroheure) [reply product_variant_default_code/README.rst:51]: You're right. But I'm currently very very busy and can't do that before january.

**Comment by andreampiovesana** [comment]
why close?

**Comment by pedrobaeza** [comment]
Comments not attended.

**Pull request closed without merging**